canvas_project/account_management/templates/register.html

```diff
@@ -5,152 +5,144 @@
     <link href="{% static 'css/login.css' %}" rel="stylesheet" type="text/css" />
 {% endblock style %}
 {% block body %}
-    <div style="display: flex; height: 100vh;">
-        <!-- Left section (form) -->
-        <div
-            <div class="d-flex flex-column justify-content-center"
-                 style="padding: 5%;
-                        min-width: 30%">
-                <h1 class="headline text-center mb-4">
-                    Join us at
-                    <br />
-                    Canvas!
-                </h1>
-                <!-- Password requirements -->
-                <div id="password-requirements" class="alert alert-info">
-                    <strong>Password must contain:</strong>
+<div class="d-flex vh-100">
+    <!-- Left section (form) -->
+    <div class="d-flex flex-column p-4 vh-100 overflow-auto" style="max-width: 400px;">
```

Comment:
```suggestion
    <div class="d-flex flex-column p-4 vh-100 overflow-auto justify-content-center" style="max-width: 400px;">
```
To center vertically